shopelectro/views/ecommerce.py

```diff
@@ -47,7 +48,20 @@ class FlushCart(ec_views.FlushCart):
 
 
 class OrderSuccess(ec_views.OrderSuccess):
-    order = Order
+    order = Order.objects.all().prefetch_related('positions')
```

Comment:
we should send only one success order, as i see

Reply:
@duker33 We still send only one success order with its positions

Reply:
@artemiy312 , y, you are right. It's DetailedView, sorry